doc/dsl/goals.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Ziele und Zustand der aktuellen DungeonDSL Implementierung"
+author: @malt-r
+---
+
+Dieses Dokument dokumentiert den aktuellen Zustand der DungeonDSL-Implementierung, also alle geplanten Features und deren aktueller Realisierungsstand.
```

Comment:
Sollte das Dokument dann eher "status.md" heissen?

Reply:
Das hat ja zwei Funktionen: einmal alle Ziele für die Implementierung im Auge zu haben und auf einen Blick den Zustand zu sehen.. also vielleicht `goals_and_status.md`?

Reply:
Ich finde kurze Dateinamen wesentlich angenehmer. 

Zudem ist der einleitende Satz in Zeile 6 recht eindeutig: Es geht um den "aktuellen Zustand", was auch geplante Dinge mit einschließt.

Insofern bin ich immer noch für `status.md`.